app/assets/stylesheets/forms.scss

```diff
@@ -114,6 +114,11 @@
   &:last-child {
     margin-bottom: 0;
   }
+
+  textarea {
+    resize: vertical;
+    min-height: 7rem;
```

Comment:
Can this be changed to a px value?

Reply:
It can, but its more connected do the height of a text element